ern-container-gen/src/generators/android/hull/lib/src/main/java/com/walmartlabs/ern/container/ElectrodeReactActivityDelegate.java

```diff
@@ -65,25 +60,26 @@ public void invokeDefaultOnBackPressed() {
     };
 
     public ElectrodeReactActivityDelegate() {
-        mReactInstanceManager = ElectrodeReactContainer.getReactInstanceManager();
+
```

Comment:
Can we get rid of this constructor as it is now useless (better than keep it empty for no reason) ?